test/csharp/csharp.tests.csproj

```diff
@@ -21,10 +21,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit" Version="2.4.0-beta.2.build4010" />
+    <PackageReference Include="Xunit.SkippableFact" Version="1.3.6" />
     <!-- DotNetCliToolReference element specifies the CLI tool that the user wants to restore in the context of the project. -->
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
-    <PackageReference Include="Xunit.SkippableFact" Version="1.3.3" />
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.4.0-beta.1.build3958" />
```

Comment:
I think we should open new tracking issue to update the beta to release.
And Microsoft.VisualStudio.Web.CodeGeneration.Tools  too.

Reply:
Created https://github.com/PowerShell/PowerShell/issues/6977